### PR TITLE
fix: design-tokens-aio-import

### PR DESCRIPTION
**Pull request by gyfchong**
Testing out resolving an issue with design-tokens importing with esm by removing the need to import.

### CODE REVIEW

File: packages/components/src/KaizenProvider/ThemeProvider/themes/types.ts [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+ * type Font = {
+ *   fontSize: `${number}rem`,
+ *
+ *  }
+ * type HexDigit = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | A | B | C | D | E | F ;
+ * type Color = `rgba(${number}, ${number}, ${number}, ${number}) | #${HexDigit}${HexDigit}${HexDigit}` // You get the point
+ *
+ * ```
+ * */
+type KaizenCSSProperties = ReactCSSProperties
```
Comment (HeartSquared) [inline packages/components/src/KaizenProvider/ThemeProvider/themes/types.ts:17]: Just realised how weird this is. We just create a bunch of aliases lol
```
type KaizenCSSProperties = ReactCSSProperties = CSSProperties
```

File: packages/components/src/KaizenProvider/ThemeProvider/themes/types.ts [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+type Hex = string
+// Once TypeScript is upgraded in the repo, you can use Lowercase<>
```
Comment (HeartSquared) [inline packages/components/src/KaizenProvider/ThemeProvider/themes/types.ts:20]: Has TS been upgraded yet? 😂

File: packages/components/src/index.ts [hunk 3]
```diff
@@ -51,8 +51,6 @@ export {
   makeCSSVariableTheme,
   mapLeafsOfObject,
 } from "@kaizen/design-tokens"
-export * from "@kaizen/design-tokens/src/themes"
-export * from "@kaizen/design-tokens/src/types"
```
Comment (HeartSquared) [inline packages/components/src/index.ts:55]: Was this referencing `themes/types.ts`? Or are we missing something?

**Review by HeartSquared — APPROVED** [review]

File: packages/components/src/index.ts (hunk 3 above)
Comment (gyfchong) [reply packages/components/src/index.ts:55]: Oh you're right, that doesn't exist haha, it should've been `types.ts` yes. Another way to tackle the issue might've been to changes these to export from `dist/themes` and not `src`. But I chose the nuclear option lol

Comment (HeartSquared) [reply packages/components/src/index.ts:55]: Perhaps exporting these from `dist` may have prevented the problem? 😂

File: packages/components/src/KaizenProvider/ThemeProvider/themes/types.ts (hunk 2 above)
Comment (gyfchong) [reply packages/components/src/KaizenProvider/ThemeProvider/themes/types.ts:20]: Lol touching this broke everything, maybe later :P

File: packages/components/src/index.ts (hunk 3 above)
Comment (gyfchong) [reply packages/components/src/index.ts:55]: We'll never know now 😈

**Pull request merged**